climada/entity/exposures/base.py

```diff
@@ -906,15 +906,29 @@ def copy(self, deep=True):
             **metadata
         )
 
-    def write_raster(self, file_name, value_name='value', scheduler=None):
+    def write_raster(self, file_name, value_name=['value'], scheduler=None):
         """Write value data into raster file with GeoTiff format
 
         Parameters
         ----------
         file_name : str
-            name output file in tif format
+            output tif file to write to
+        value_name : str or list of str
+            column(s) in the geodataframe containing data. If multiple values are provided, each column will be
+            written to a separate band in the output raster
+        scheduler : str, optional
+            used for dask map_partitions.
+            “threads”, “synchronous” or “processes”
         """
-        if self.meta and self.meta['height'] * self.meta['width'] == len(self.gdf):
+        if not isinstance(value_name, list):
+            value_name = [value_name]
+
```

Comment:
I would add the correct file ending by default:

`file_name = Path(file_name).with_suffix('.tiff')`

Although, this might be something to be added to the `write_raster` method rather than here.

Reply:
Yeah it would be better in `write_raster`.
But would anyone ever provide a file name without an extension? I can't imagine that.

Unless this is to enforce `.tiff` extensions over `.tif`, in which case, why?

Reply:
I do. Also, it is a simple thing to do :).

Reply:
Implemented in 2553a0e8